frontend/src/metabase/search/components/InfoText/InfoText.tsx

```diff
@@ -26,6 +26,7 @@ import { LastEditedInfoText, LastEditedInfoTooltip } from "./InfoText.styled";
 export type InfoTextProps = {
   result: WrappedResult;
   isCompact?: boolean;
+  isEnabled?: boolean;
```

Comment:
1. Using `disabled` would be more aligned with [DOM nomenclature](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled).
2. `InfoTextEditedInfo` does not make use of that prop, yet it uses this very interfece. It's misleading, as it suggests it does support that prop. Similarly, `isCompact` is used in only one of the components here. This would not be a problem if each component lived in a separate file and had its own explicit `Props` definition.

Reply:
I don't think using `disabled` makes sense here - `disabled` usually implies that there's a clickable element that's grayed out (like a disabled link) but here we just want some text.

On the `InfoTextEditedInfo` thing we can do that in a separate PR specifically for that - let's not muddy this PR with extra things. I can make the changes to the props here though.

Reply:
Ok split it a little bit

Reply:
> I don't think using `disabled` makes sense here - `disabled` usually implies that there's a clickable element that's grayed out (like a disabled link) but here we just want some text.

In that case, given that `isEnabled` is semantically an inverse of `disabled`, should we call it `showLinks`?

Reply:
sure, why not

Reply:
ok changed @kamilmielnik